cli/main.c

```diff
@@ -1,6 +1,6 @@
 /*
  * Microsemi Switchtec(tm) PCIe Management Command Line Interface
- * Copyright (c) 2017, Microsemi Corporation
+ * Copyright (c) 2020, Microsemi Corporation
```

Comment:
I'm not sure if we need to update this. It seems the year here should be the "first year of the publication". Comment?

Reply:
Yes, it's really meant to be first year of publication. Though the year isn't that important, we have git logs to show when things were first published.

More importantly, we don't need a bunch of churn in every commit changing every year.

Reply:
I did this since for FW it's always been a requirement to update the copyright date whenever we touch a file. If that's not a requirement for this project then I'm good with that -- I agree that it's nice to avoid the churn.